org.kde.klevernotes.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "org.kde.klevernotes",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
+    "sdk": "org.kde.Sdk",
+    "base": "io.qt.qtwebengine.BaseApp",
+    "base-version": "6.6",
+    "command": "klevernotes",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
```

Comment:
why is home needed ?

Reply:
I can't switch/create klevernotes storage if I remove this. 
Do you suggest using a different filesystem than `home` ?

Reply:
what is "klevernotes storage" and where is it created ?

Reply:
It's just a folder. The app need to be able to create and read from a folders/files. The user usually choose the location, that's why I choose `home` snce there's no predefined location.

Reply:
The location should be chosen via the file portal where the user would select the directory and then the app would get access to it. I don't think it needs full home access.